kitsune/kpi/models.py

```diff
@@ -16,6 +16,12 @@
 EXIT_SURVEY_NO_CODE = 'exit-survey:no'
 EXIT_SURVEY_DONT_KNOW_CODE = 'exit-survey:dont-know'
 
+CONTRIBUTOR_COHORT_CODE = 'contributor'
+KB_CONTRIBUTOR_COHORT_CODE = 'contributor:kb'
+KB_L10N_CONTRIBUTOR_COHORT_CODE = 'contributor:kb:l10n'
+SUPPORT_FORUM_HELPER_COHORT_CODE = 'contributor:support_forum:helper'
+AOA_CONTRIBUTOR_COHORT_CODE = 'contributor:aoa'
```

Comment:
yeah true, I should probably also rename it to `KB_ENUS_CONTRIBUTOR_COHORT_CODE`